requirements.txt

```diff
@@ -1,7 +1,7 @@
 alembic==1.4.2
 canonicalwebteam.flask_base==0.6.3
 canonicalwebteam.http==1.0.3
-canonicalwebteam.blog==6.0.0
+git+git://github.com/albertkol/canonicalwebteam.blog.git@re-enable-cloudinary#egg=canonicalwebteam.blog
```

Comment:
can you try `git+https` instead?

Reply:
why? is there a problem with using SSH?

Reply:
The image was not building, because it did not have `git` as a dependency in the python step. Just wanted to try this to check if it avoided `pip` attempting to git clone. It does not so we added `git` in the dockerfile.